examples/chaincode/go/asset_management/asset_management.go

```diff
@@ -22,6 +22,8 @@ import (
 	"github.com/hyperledger/fabric/core/chaincode/shim"
```

Comment:
Have you run goimports on this file? When I run it, it inserts a newline after line 21 which seems to be missing here.